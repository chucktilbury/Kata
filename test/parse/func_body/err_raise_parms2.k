;-------------------------------------------------------------------------------
; Raise expects a symbol and a string literal
;-------------------------------------------------------------------------------

func name()() {
	raise(123, "string")
	}
