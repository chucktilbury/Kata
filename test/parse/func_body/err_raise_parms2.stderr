Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_raise_parms2.k: 6: 11: expected a SYMBOL but got literal signed
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_raise_parms2.k: 8: 1: expected function body element but got end of file
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_raise_parms2.k: 8: 1: expected function body but got end of file
